package-lock.json

```diff
@@ -0,0 +1,6148 @@
+{
```

Comment:
Our policy is to not commit these as per the current .gitignore in trunk - https://github.com/fluid-project/kettle/blob/master/.gitignore#L3 - perhaps this branch needs merging up?

Reply:
It was merged up, but there a commit of `package-lock.json` from before the current .gitignore - I've removed it.